cloudinit/net/activators.py

```diff
@@ -15,6 +15,19 @@
 LOG = logging.getLogger(__name__)
 
 
+def _alter_interface(cmd, device_name):
```

Comment:
Might as well type hint it too
```suggestion
def _alter_interface(cmd, device_name) -> bool:
```